src/mmw/js/src/compare/utils.js

```diff
@@ -57,8 +58,33 @@ function mapScenariosToHydrologyTableData(scenarios) {
             };
         });
 }
+/**
+ * Returns a list of selections for water quality, filtered down to
+ * only those that are present in all scenarios. This allows for
+ * opening the Compare View when there are old scenarios with
+ * "Other Upland Areas", and new scenarios with "Low-Density Open Space".
+ * In this case, neither of these will be shown in the Compare View.
+ *
+ * @param scenarios ScenarioCollection
+ */
+function getQualitySelections(scenarios) {
+    return _.filter(constants.gwlfeQualitySelectionOptionConfig,
+                    function(selection) {
+        return scenarios.every(function(scenario) {
+            var result = scenario
+                         .get('results')
+                         .findWhere({ name: 'quality' })
+                         .get('result');
+
+            return _.some(result[selection.group], function(load) {
```

Comment:
Totally fine to leave as is, but since `.every` is used above, this implementation could potentially also replace `_.some` and `_.filter` with the vanilla JS versions.

Reply:
`scenarios` is a collection, not an array, so that `every` is actually using `underscore`'s equivalent. Thus, they are all using libraries.